### DISCUSSION TITLE
Trouble with argon

### DISCUSSION THREAD

**Discussion by jschindler89** [brucefan1983/GPUMD#638] [Q&A]
Hi,

I am playing around with LJ argon system at 10 K. 

Using the files attached, I see that the max frequency for the DOS is about 4 THz which is twice as large as what we would expect for crystal LJ argon at low temperatures. Would you know how I can fix this?
[Ar.zip](https://github.com/user-attachments/files/15551441/Ar.zip)

Jack

**Comment by brucefan1983** [comment]
Could you also provide the model.xyz file such that I can reproduce the results?

**Reply by jschindler89** [reply]
Here it is! Thank you.
[model.xyz.zip](https://github.com/user-attachments/files/15585212/model.xyz.zip)

**Reply by jschindler89** [reply]
Also, the thermal conductivity seems to be ~ 2 times higher than literature. Thank you for your help. Jack

**Comment by brucefan1983** [comment]
Because you have set the mass to 12 in model.xyz. It should be 40. You can
remove the mass item. GPUMD has a default mass for any element
taken from a CRC handbook.

**Reply by jschindler89** [reply]
Thank you so much! Jack.

**Reply by jschindler89** [reply]
Hi Bruce, 

Is there a way to shift up the LJ potential at a designated cutoff?

**Comment by brucefan1983** [comment]
You can modify the code by yourself :-)
A couple of lines.

**Reply by jschindler89** [reply]
Sounds good. I will try. 

To know where to start, I need to change lj.cu in force folder, right?

**Reply by brucefan1983** [reply]
Yes, to modify this function (perhaps you need to pass the cutoff in and use it):
```c++
// get U_ij and (d U_ij / d r_ij) / r_ij (the LJ potential)
static __device__ void
find_p2_and_f2(double s6e4, double s12e4, double d12sq, double& p2, double& f2)
{
  double d12inv2 = 1.0 / d12sq;
  double d12inv6 = d12inv2 * d12inv2 * d12inv2;
  f2 = 6.0 * (s6e4 * d12inv6 - s12e4 * 2.0 * d12inv6 * d12inv6) * d12inv2;
  p2 = s12e4 * d12inv6 * d12inv6 - s6e4 * d12inv6;
}
```

**Reply by jschindler89** [reply]
Thank you!
